### PR TITLE
updated readme file with the daily progress

### PR DESCRIPTION
**Pull request by Siianko**
### Our daily progress

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,14 @@
...
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
```
Comment (faraznaeem) [inline README.md:24]: So these points cover pretty much the things that you need to update the README with, rather what you have done during the day

Comment (tochman) [reply README.md:24]: Will this be addressed?

Comment (Andycutter) [reply README.md:24]: I tried to with my readme PR, but I seem to get all the changes in the other files with, even though I pull from upstream development and only edit the README

Comment (Andycutter) [reply README.md:24]: Made new PR with changes to README
https://github.com/CraftAcademy/cibus/pull/11

**Pull request closed without merging**